lib/screens/Profile.tsx

```diff
@@ -97,6 +97,22 @@ class UserProfile extends React.Component<UserProfileProps, UserProfileState> {
     }
   }
 
+  showSwitchModal() {
```

Comment:
Is the plan to show this every time?

Reply:
Just placeholder plumbing for the moment so I can make sure it’s all working after screen refactor. An action sheet is better but is iOS only so I might make a simple popover menu.